### DISCUSSION TITLE
Alaska STILL not reporting?

### DISCUSSION THREAD

**Discussion by GrizzlyAK** [weather-gov/api#284] [Q&A]
Hi, I'm trying to use the API to gather forecast information for Eagle River, AK (Anchorage) in Home Assistant, but am getting an error. When I put in the following URL into my browser directly

https://api.weather.gov/points/61.3142,-149.4848

everything from gridx: to observationStations: is "null". If is use the example lat/long for Washington, I get valid URLs that then provide the forecast data. Are you reporting forecast data for Alaska? I previously posted about this bug, but it was closed but not resolved (#51) Thanks. Shane

Here's the output from the above URL:

{
    "@context": [
        "https://geojson.org/geojson-ld/geojson-context.jsonld",
        {
            "@version": "1.1",
            "wx": "https://api.weather.gov/ontology#",
            "s": "https://schema.org/",
            "geo": "http://www.opengis.net/ont/geosparql#",
            "unit": "http://codes.wmo.int/common/unit/",
            "@vocab": "https://api.weather.gov/ontology#",
            "geometry": {
                "@id": "s:GeoCoordinates",
                "@type": "geo:wktLiteral"
            },
            "city": "s:addressLocality",
            "state": "s:addressRegion",
            "distance": {
                "@id": "s:Distance",
                "@type": "s:QuantitativeValue"
            },
            "bearing": {
                "@type": "s:QuantitativeValue"
            },
            "value": {
                "@id": "s:value"
            },
            "unitCode": {
                "@id": "s:unitCode",
                "@type": "@id"
            },
            "forecastOffice": {
                "@type": "@id"
            },
            "forecastGridData": {
                "@type": "@id"
            },
            "publicZone": {
                "@type": "@id"
            },
            "county": {
                "@type": "@id"
            }
        }
    ],
    "id": "https://api.weather.gov/points/61.3142,-149.4848",
    "type": "Feature",
    "geometry": {
        "type": "Point",
        "coordinates": [
            -149.48480000000001,
            61.3142
        ]
    },
    "properties": {
        "@id": "https://api.weather.gov/points/61.3142,-149.4848",
        "@type": "wx:Point",
        "cwa": "AER",
        "forecastOffice": "https://api.weather.gov/offices/AER",
        "gridId": null,
        "gridX": null,
        "gridY": null,
        "forecast": null,
        "forecastHourly": null,
        "forecastGridData": null,
        "observationStations": null,
        "relativeLocation": {
            "type": "Feature",
            "geometry": {
                "type": "Point",
                "coordinates": [
                    -149.28432900000001,
                    61.174250000000001
                ]
            },
            "properties": {
                "city": "Anchorage",
                "state": "AK",
                "distance": {
                    "value": 18898.914340104999,
                    "unitCode": "unit:m"
                },
                "bearing": {
                    "value": 325,
                    "unitCode": "unit:degrees_true"
                }
            }
        },
        "forecastZone": "https://api.weather.gov/zones/forecast/AKZ101",
        "county": "https://api.weather.gov/zones/county/AKC020",
        "fireWeatherZone": "https://api.weather.gov/zones/fire/AKZ101",
        "timeZone": "America/Anchorage",
        "radarStation": "PAHG"
    }
}

**Comment by scadergit** [comment]
The issue involves a core service that cannot be repaired, only refactored. Management recognizes the importance of that service. If not next it is near the top of the backlog. We're working as quickly as possible to stablize code that was handed to us, but we're a small team maintaining several applications and services. Once we get through the current effort there were be more focus on the API again. Sorry that doesn't provide a specific date, but it will work far better when we are allowed to fix it.

**Comment by GrizzlyAK** [comment]
OK, thanks for the response Jared. I understand. Cheers.

 

 

All truths are easy to understand once they are discovered. The point is to discover them.

-- Galileo
